Android crash java.lang.ClassNotFoundException

Android apps which do not contain the `com.google.android.gms:play-services-ads-identifier` package crash with the following error:

```
Caused by: java.lang.NoClassDefFoundError: Failed resolution of: Lcom/google/android/gms/ads/identifier/AdvertisingIdClient;
at pro.piwik.sdk.Tracker$1.doInBackground(Tracker.java:225)
at pro.piwik.sdk.Tracker$1.doInBackground(Tracker.java:220)
at android.os.AsyncTask$3.call(AsyncTask.java:394)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
```

This is due to the fact that the upstream library doesn't mark its dependencies properly. See https://github.com/PiwikPRO/PPMS-PublicDocs/issues/520

This could probably be circumvented by adding the upstream dependency to this library.

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -127,5 +127,6 @@ dependencies {
   // noinspection GradleDynamicVersion
   api 'com.facebook.react:react-native:+'
   api 'pro.piwik.sdk:piwik-sdk:1.+'
+  implementation "com.google.android.gms:play-services-ads-identifier:+"
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
 }

